devdox/app/services/api_keys_service.py

```
@@ -0,0 +1,119 @@
+
+        return cls(
+            api_key_store=api_key_store,
+            api_key_manager=api_key_manager,
+        )
+
+    async def generate_api_key(self, user_claims: UserClaims):
+
+        max_generation_attempts = 6
+
+        result = None
+        for attempt in range(1, max_generation_attempts + 1):
```

Ruff (0.11.9)

Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)